src/api/spec/features/webui/requests_spec.rb

```diff
@@ -101,6 +105,11 @@
         click_link 'Request role addition'
         within('#add-role-modal') do
           find(:id, 'role').select('Maintainer')
+          find(:id, 'role_type').select('User')
```

Comment:
@quatran hey, the PR looks good to me so far, I have one suggestion. You already adjusted this feature test for the request role addition of a User. Maybe you could take over this logic and do the same for the group :)  If you need help, let me know. But first step would be to see how its done here for the User.